qtconsole/console_widget.py

```diff
@@ -21,6 +21,8 @@
 
 from traitlets.config.configurable import LoggingConfigurable
 from traitlets import Bool, Enum, Integer, Unicode
+from traitlets import HasTraits, Unicode, observe
+
```

Comment:
Need to import `default` for the suggestion about the `ShortcutManager` class definition works:
```suggestion
from traitlets import default, HasTraits, Unicode, observe
```